lib/winston/create-logger.js

```diff
@@ -75,7 +75,9 @@ module.exports = function (opts = {}) {
       // Optimize the hot-path which is the single object.
       if (args.length === 1) {
         const [msg] = args;
-        const info = msg && msg.message && msg || { message: msg };
+        // NOTE: If `msg` is an Object, clone it so changes made, such as the addition of the level field,
+        // do not affect the original object.
+        const info = msg && msg.message && { ...msg } || { message: msg };
```

Comment:
[structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) can solve this issue assuming support for it aligns with Winston's minimum supported platforms.